Dockerfile-alpine.template

```diff
@@ -69,6 +69,12 @@ RUN set -eux; \
 	patch -p1 -i thread-stack-fix.patch; \
 	rm thread-stack-fix.patch; \
 	\
+# https://bugs.ruby-lang.org/issues/17723
```

Comment:
```suggestion
# https://bugs.ruby-lang.org/issues/17723 (autoconf-2.70.patch)
```

Should include the sed key so that the comment gets deleted and drop the extra `\` line so that the end result doesn't get two "empty" lines when there is no patch. (It's logically another patch, so chose to drop the top and keep the bottom empty line.)